### ISSUE TITLE
Babel Error: `@babel/plugin-proposal-private-property-in-object ` explicit dependency

### ISSUE THREAD

**Issue by j3tan** [milesj/packemon#209]
Hitting a newly introduced error message in `babel` version `7.22.5` causes the following error:

```sh
--- PLACEHOLDER PACKAGE ---
This @babel/plugin-proposal-private-property-in-object version is not meant to
be imported. Something is importing
@babel/plugi-proposal-private-property-in-object without declaring it in its
dependencies (or devDependencies) in the package.json file.
Add "@babel/plugin-proposal-private-property-in-object" to your devDependencies
to work around this error. This will make this message go away.
```

Since `packemon` uses `@babel/plugin-proposal-private-property-in-object` for babel config, it needs to explicitly declare it as a dependency.
https://github.com/milesj/packemon/blob/76085236fb1b246ae8a0c1e468737910cee6976d/packages/packemon/src/babel/config.ts#L140

Related:
https://github.com/babel/babel-plugin-proposal-private-property-in-object
https://github.com/babel/babel/pull/15687
https://github.com/babel/babel/issues/15694

**Comment by milesj** [comment]
@j3tan This is a weird one. I'm not importing `@babel/plugin-proposal-private-property-in-object` anywhere.

But I am importing `@babel/plugin-transform-private-property-in-object` which is different (note transform vs proposal). Maybe node modules in a weird state? Does deduping fix it? 

I'll dig a bit deeper anyways.

**Comment by milesj** [comment]
I can't seem to reproduce this. Is it just being logged to the console during a build?

**Comment by j3tan** [comment]
@milesj My bad, this may be due to using an older version of `packemon` . I had not realized you moved away from the proposal package 2 weeks ago and we definitely haven't bumped since before that.

I'll verify and close if that is the case

**Comment by j3tan** [comment]
Some repos are still using packemon@2.x so we're using yarn resolutions to bypass the warning for now.

**Comment by milesj** [comment]
Yea this was fixed in 3.1.1. I'll leave this open for now in case there's more issues around it.

**Comment by MB-Prabhu** [comment]
how to solve this error im a beginner id dont know much technical terms, can anyone tell me how to solve in simple words??!!
